src/seedu/addressbook/commands/Command.java

```diff
@@ -43,6 +43,13 @@ public CommandResult execute(){
         throw new UnsupportedOperationException("This method should be implement in child classes");
     }
 
+    /**
+     * Executes the command if command mutates data and returns the result.
+     */
+    public boolean isMutating(){
+        return false;
```

Comment:
as discussed during the tutorial, it is not recommended to include default behavior.

Also, you've missed out the space before left curly